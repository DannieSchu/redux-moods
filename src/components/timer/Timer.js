import React from 'react';
import PropTypes from 'prop-types';

const Timer = ({ time }) => <h2>Time remaining: {time}</h2>;

Timer.propTypes = {
  time: PropTypes.number.isRequired
};

export default Timer;

