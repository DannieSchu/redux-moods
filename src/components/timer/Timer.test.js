import React from 'react';
import { shallow } from 'enzyme';
import Timer from './Timer';

describe('Timer component', () => {
  it('renders Timer', () => {
    const wrapper = shallow(<Timer time={60} />);
    expect(wrapper).toMatchSnapshot();
  });
});

