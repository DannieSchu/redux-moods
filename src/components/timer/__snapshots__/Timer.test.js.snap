// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timer component renders Timer 1`] = `
<h2>
  Time remaining: 
  60
</h2>
`;
